### ISSUE TITLE
Running Ohai in appveyor takes nearly 2 mins on Windows

### ISSUE THREAD

**Issue by jaym** [chef/ohai#715]
Running Ohai 8.9 is extremely slow on Windows. In appveyor, it takes nearly 2 mins. This seems to have been caused by the addition of the package plugin. Attached the run times for each plugin below:

```
       user     system      total        real
CPU  0.000000   0.000000   0.000000 (  1.040844)
Filesystem  0.031000   0.016000   0.047000 (  0.125003)
Kernel  0.032000   0.031000   0.063000 (  0.078118)
Drivers  0.453000   0.250000   0.703000 (  1.570801)
Memory  0.000000   0.000000   0.000000 (  0.015623)
Network  2.687000   1.844000   4.531000 (  5.109379)
NetworkRoutes  0.000000   0.000000   0.000000 (  0.000000)
NetworkListeners  0.000000   0.000000   0.000000 (  0.000000)
NetworkAddresses  0.000000   0.000000   0.000000 (  0.000000)
OS  0.000000   0.000000   0.000000 (  0.000000)
Platform  0.000000   0.000000   0.000000 (  0.000000)
Uptime  0.000000   0.000000   0.000000 (  2.402302)
Virtualization  0.000000   0.000000   0.000000 (  0.015623)
Joyent  0.000000   0.000000   0.000000 (  0.000000)
VirtualizationInfo  0.000000   0.000000   0.000000 (  0.000000)
Azure  0.000000   0.000000   0.000000 (  0.000000)
C  0.047000   0.062000   0.109000 (  0.312501)
Elixir  0.016000   0.000000   0.016000 (  0.031249)
Erlang  0.000000   0.000000   0.000000 (  0.093751)
Go  0.015000   0.000000   0.015000 (  0.300237)
Groovy  0.000000   0.016000   0.016000 (  0.031249)
Java  0.000000   0.000000   0.000000 (  0.657026)
Languages  0.000000   0.000000   0.000000 (  0.000000)
Lua  0.016000   0.000000   0.016000 (  0.031252)
Mono  0.000000   0.000000   0.000000 (  0.031248)
Nodejs  0.016000   0.000000   0.016000 (  0.093754)
Perl  0.000000   0.000000   0.000000 (  0.140620)
PHP  0.000000   0.015000   0.015000 (  0.031250)
Powershell  0.000000   0.000000   0.000000 (  2.382702)
Python  0.000000   0.000000   0.000000 (  0.272729)
Ruby  0.000000   0.000000   0.000000 (  2.578125)
Rust  0.015000   0.000000   0.015000 (  0.046875)
Chef  0.000000   0.000000   0.000000 (  0.000000)
Ohai  0.000000   0.000000   0.000000 (  0.000000)
Cloud  0.000000   0.016000   0.016000 (  0.046874)
CloudV2  0.000000   0.000000   0.000000 (  0.000000)
Command  0.000000   0.000000   0.000000 (  0.000000)
PS  0.000000   0.000000   0.000000 (  0.000000)
Filesystem2  0.000000   0.000000   0.000000 (  0.000000)
SystemProfile  0.000000   0.000000   0.000000 (  0.000000)
DigitalOcean  0.000000   0.000000   0.000000 (  0.000000)
DMI  0.000000   0.000000   0.000000 (  0.031250)
EC2  0.000000   0.000000   0.000000 (  0.000000)
Eucalyptus  0.000000   0.000000   0.000000 (  0.000000)
GCE  0.000000   0.000000   0.000000 (  0.000000)
Hostname  0.016000   0.000000   0.016000 (  1.361591)
InitPackage  0.000000   0.000000   0.000000 (  0.000000)
IpScopes  0.000000   0.000000   0.000000 (  0.000000)
Keys  0.000000   0.000000   0.000000 (  0.000000)
SSHHostKey  0.000000   0.000000   0.000000 (  0.000000)
Linode  0.000000   0.000000   0.000000 (  0.000000)
BlockDevice  0.000000   0.000000   0.000000 (  0.000000)
LSB  0.000000   0.000000   0.000000 (  0.000000)
Mdadm  0.000000   0.000000   0.000000 (  0.000000)
OhaiTime  0.000000   0.000000   0.000000 (  0.000000)
Openstack  0.000000   0.000000   0.000000 (  0.000000)
Packages  2.047000   1.328000   3.375000 ( 92.044243)
Passwd  0.000000   0.000000   0.000000 (  0.000000)
Rackspace  0.000000   0.000000   0.000000 (  0.000000)
RootGroup  0.000000   0.000000   0.000000 (  0.000000)
Softlayer  0.000000   0.000000   0.000000 (  0.000000)
Zpools  0.000000   0.000000   0.000000 (  0.000000)
Virtualbox  0.000000   0.000000   0.000000 (  0.296499)
VMware  0.000000   0.000000   0.000000 (  0.000000)
```

cc @adamedx @danielsdeleo

**Comment by danielsdeleo** [comment]
Did we just introduce the package plugin in this release? Thoughts about removing it?

**Comment by tas50** [comment]
The package plugin is new for this release so it hasn't actually shipped to anyone.

**Comment by smurawski** [comment]
There are a bunch of folks doing what the package plugin does in their recipes and for the normal use of chef-client, having that pulled in by ohai at the start of the run is a better workflow.

**Comment by danielsdeleo** [comment]
Is that worth the extra 90s for everyone?

**Comment by smurawski** [comment]
@danielsdeleo I think for most server workloads it won't be anywhere near 90 seconds.. the appveyor build workers have an insane amount of software installed on them for a server (to run just about any type of common windows software build).

**Comment by jaym** [comment]
90s seconds seems like a worst case. On my box it adds 10 seconds, which IMO is still a lot. If we want to include this, we need to make it something that does not run by default. We can discuss ways to make this seamless for users, but I do not think the added time is acceptable.

**Comment by lamont-granquist** [comment]
IMO back it out until we at least get default-disabled plugins and an enabled_plugins config option.

**Comment by danielsdeleo** [comment]
Yeah, I'm in the process of removing the pnp plugin because it's slow and most folks don't need the data. For now I'm extracting it to a cookbook, which could be a general  "windows-ohai-extras" thing where we could also put the package plugin until we make it much easier to handle plugins that are useful but have bad tradeoffs for folks who don't need them.

**Comment by jaym** [comment]
fixed by #717
